richtextfx/src/main/java/org/fxmisc/richtext/model/SimpleEditableStyledDocument.java

```diff
@@ -22,14 +22,14 @@
 /**
  * Provides an implementation of {@link EditableStyledDocument}
  */
-public final class SimpleEditableStyledDocument<PS, S> implements EditableStyledDocument<PS, S> {
+public final class SimpleEditableStyledDocument<PS, SEG, S> implements EditableStyledDocument<PS, SEG, S> {
```

Comment:
SimpeEditableStyledTextDocument, probably - inherited from GenericEditableStyledDocument ...
I was thinking the same already, let me see what consequences that would have ;-)